pkgs/top-level/python-packages.nix

```diff
@@ -18901,6 +18901,8 @@ in {
      };
   };
 
+  proselint = callPackage ../tools/text/proselint { };
```

Comment:
@7c6f434c thanks for checking old PR's.

@alibabzo if this is an application, then it shouldn't be called via `python-packages.nix` but directly from `all-packages.nix`.